CsharpExample/CollectionsReport/Results.csv

```diff
@@ -0,0 +1,302 @@
+7d9bd546-b15b-40b6-96b9-6e6b54aaec16,05/21/2021 00:00:00,3923,Carina
+38812ded-d0f2-42fb-8aab-5f4618154d03,05/21/2021 00:00:00,836,Sakae
+40452b14-d3a5-460a-ac10-bb84e4c07473,05/21/2021 00:00:00,2075,Carlos
+e93ebc96-e0bd-4812-ba70-f5d7211d06b1,05/21/2021 00:00:00,2928,Shane
+1d1116eb-ea76-449a-94cd-aaf65ce2c210,05/21/2021 00:00:00,1950,Stephen
+39f9bacf-c14e-4c0f-a283-bcc35ab50f7a,06/05/2021 00:00:00,3886,Dakota
+edb79d33-b5cc-42bf-a610-69f1d52674f3,06/05/2021 00:00:00,917,Loc
+5d293bdb-804b-417d-acc2-553cd900b184,06/05/2021 00:00:00,1466,Carina
+ed78ea3a-739f-4e53-a7de-4e48f0c5a58a,06/05/2021 00:00:00,4855,Amin
+1bea0334-c3a6-4d27-823d-a4a8a4dc91a2,06/05/2021 00:00:00,3090,Shane
+e318cc0a-fe79-46f8-9eb6-2abde5d4785b,06/15/2021 00:00:00,4594,Gibby
+de5393e7-8740-4f5b-a423-8f89ef2f8b43,06/15/2021 00:00:00,1293,Kelley
+4762850e-50d0-41fa-a72a-ddde25f3af55,06/15/2021 00:00:00,664,Carina
+91fbdc58-34ea-4cba-9527-5f2c3407da73,06/15/2021 00:00:00,4487,Conrad
+55a60d65-1128-4f79-9e06-1f5314c485d9,06/15/2021 00:00:00,3768,Loc
+4b0c8edc-f9c6-4c72-a8e9-17d9b8f91f06,06/15/2021 00:00:00,3933,Ted
+90ae0658-f948-4fc8-ab61-ed3990699aae,06/15/2021 00:00:00,2949,Ryan
+d031aabb-e75a-443f-aa14-1cb87cb9137e,06/15/2021 00:00:00,3534,Gibby
+078f2f7e-30d6-42f5-8686-9bb3c130a97d,06/15/2021 00:00:00,4494,Sakae
+dbc7e625-958f-46ff-9220-e597837589ea,06/15/2021 00:00:00,1192,Carlos
+8d3a8b38-954a-4051-a6b5-c9033db180bf,06/15/2021 00:00:00,4853,Ted
+35d9f320-d284-4d9a-a8c8-cfb0582f2aaa,06/15/2021 00:00:00,4267,Andrew
+33fc00cc-6f1a-4063-8678-14ae3aac7696,06/15/2021 00:00:00,2589,Catherine
+1408a935-7973-4230-b17b-c8400b27f685,06/15/2021 00:00:00,4783,Dakota
+7f7f75a2-4581-4baa-955d-45b3054c5938,07/05/2021 00:00:00,511,Dakota
+d9e27203-4fda-46a1-b5dd-4ac174b3b230,07/05/2021 00:00:00,1254,Shane
+8aef1afa-5036-464d-89c7-6b12ee5d95b9,07/05/2021 00:00:00,3751,Carlos
+fe6d1a16-fd0b-488a-859b-9e60da2a1cf2,07/05/2021 00:00:00,2723,Marisol
+6222f7c0-f73d-4221-afcc-6624fb7796d1,07/05/2021 00:00:00,3702,Catherine
+dacf615c-25d8-4378-9a03-89d4d7cb9135,07/05/2021 00:00:00,3470,Kelley
+2b52bbd6-cfa6-4797-a04e-d7058e021d23,07/05/2021 00:00:00,2646,Loc
+d5a9462b-aa11-4370-b0a3-a864322e36d1,07/05/2021 00:00:00,2248,Shane
+8aef1afa-5036-464d-89c7-6b12ee5d95b9,07/05/2021 00:00:00,3751,Carlos
+d6b8c0e5-7b2c-41d5-98d5-f6f1663c0ac8,07/10/2021 00:00:00,3587,Carlos
+f7dcf7e1-979e-4bea-bb69-277c67c00e3a,07/10/2021 00:00:00,3860,Gibby
+7d75fb17-d3f5-44df-a9bd-6e414e43bb9c,07/10/2021 00:00:00,3292,Catherine
+5c5953b9-bce1-4e2f-aa47-a85005db0020,07/10/2021 00:00:00,2100,Marisol
+010d3990-1b24-4bd0-8c5c-7a3d2ac8d43d,07/10/2021 00:00:00,1807,Max
+7ecb7613-5f41-4fa7-aa8c-a95ad8c5ee46,07/10/2021 00:00:00,2246,Catherine
+d74565f3-957e-4a3b-bd91-510d23dc505a,07/10/2021 00:00:00,3900,Ted
+a3487ab9-6c5e-4121-ae83-1f8c69906290,07/10/2021 00:00:00,1474,Stephen
+73a9e127-4a14-4836-afb1-a300860f0959,07/10/2021 00:00:00,645,Shane
+e949b9ea-6e56-43d9-a5b1-63868b20eb15,07/10/2021 00:00:00,1214,Shane
```

Comment:
Let's leave this file out of the PR.  It's just demo data.

Reply:
Sounds good!